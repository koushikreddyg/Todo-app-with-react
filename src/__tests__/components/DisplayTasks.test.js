import React from 'react';
import {shallow} from 'enzyme';
import DisplayTasks from '../../components/DisplayTasks';
import Tasks from '../../fixtures/Tasks';
test('this is test case for display tasks',()=>{
    const wrapper=shallow(<DisplayTasks Tasks={Tasks}/>);
    expect(wrapper).toMatchSnapshot();
})
