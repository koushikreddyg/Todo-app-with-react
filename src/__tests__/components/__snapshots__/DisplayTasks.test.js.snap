// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`this is test case for display tasks 1`] = `
<div>
  <div
    key="24"
  >
    <h4>
      <Checkbox
        indeterminate={false}
        onChange={[Function]}
        prefixCls="ant-checkbox"
      >
        <Link
          replace={false}
          to="/edit/24"
        >
          Cricket
           
        </Link>
      </Checkbox>
    </h4>
  </div>
  <div
    key="25"
  >
    <h4>
      <Checkbox
        indeterminate={false}
        onChange={[Function]}
        prefixCls="ant-checkbox"
      >
        <Link
          replace={false}
          to="/edit/25"
        >
          Football
           
        </Link>
      </Checkbox>
    </h4>
  </div>
  <div
    key="26"
  >
    <h4>
      <Checkbox
        indeterminate={false}
        onChange={[Function]}
        prefixCls="ant-checkbox"
      >
        <Link
          replace={false}
          to="/edit/26"
        >
          Hockey
           
        </Link>
      </Checkbox>
    </h4>
  </div>
</div>
`;
